benchmarks/nx-cugraph/pytest-based/bench_algos.py

```diff
@@ -489,7 +502,7 @@ def bench_shortest_path(benchmark, graph_obj, backend_wrapper):
     node = get_highest_degree_node(graph_obj)
 
     result = benchmark.pedantic(
-        target=backend_wrapper(nx.shortest_path),
+        target=backend_wrapper(nx.shortest_path, force_unlazy_eval=True),
```

Comment:
I think it would be nice to be able to run this benchmark with both `force_unlazy_eval=True` and `force_unlazy_eval=False`.

Reply:
> I think it would be nice to be able to run this benchmark with both `force_unlazy_eval=True` and `force_unlazy_eval=False`.

Yes I agree that's useful.  I'll add a `FIXME`/issue to parameterize that.  FWIW, here's the results with/without when I ran locally:

![image](https://github.com/user-attachments/assets/edeed4ad-db2e-4c40-8787-12428ab761bd)

Fortunately run times don't have a noticeable change until the hollywood (57M edges) or larger datasets.

Reply:
https://github.com/rapidsai/nx-cugraph/issues/44